R/Flag.R

```diff
@@ -8,7 +8,7 @@
 #'
 #' @details
 #' This function provides a generalized framework for flagging sites as part of the
-#' [GSM data pipeline](https://gilead-biostats.github.io/gsm/articles/DataPipeline.html).
+#' [GSM data model](https://gilead-biostats.github.io/gsm/articles/DataModel.html).
```

Comment:
```suggestion
#' GSM data model (see `vignette("DataModel")`).
```
(See vignette reviews below for notes and links about how R Packages says this will work)